src/content/docs/security/security-privacy/compliance/certificates-standards-regulations/hipaa.mdx

```diff
@@ -138,7 +138,21 @@ The following services are not covered under the BAA:
       </td>
 
       <td>
-        Pixie: Auto-telemetry with Pixie
+        Log patterns
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        N/A
+      </td>
+
+      <td>
+        N/A
```

Comment:
@jxainguyen should this "N/A" in the second column be this: AWS, Azure, and GCP?

If so, we'd want to remove the parenthetical (AWS, Azure, and GCP) from the third column, right?